src/View/Helper/I18nHelper.php

```diff
@@ -91,4 +101,85 @@ public function changeUrlLang($newLang) : string
 
         return $url;
     }
+
+    /**
+     * Translate object field
+     * Return translation (by response object and included data, field and language)
+     *
+     * @param array $object The object to translate
+     * @param array $included The included translations data
+     * @param string $attribute The attribute name
+     * @param string $lang The lang (2 chars string)
+     * @param bool $defaultNull Pass true when you want null as default, on missing translation
+     * @return string|null
+     */
+    public function field(array $object, array $included, string $attribute, string $lang, bool $defaultNull = false) : ?string
+    {
+        $defaultValue = null;
+        if (!$defaultNull) {
+            $defaultValue = Hash::get($object, sprintf('attributes.%s', $attribute));
+        }
+        $returnValue = $this->getTranslatedField($object, $included, $attribute, $lang);
+        if ($returnValue === null) {
+            return $defaultValue;
+        }
+
+        return $returnValue;
+    }
+
+    /**
+     * Verify that object has translation for the specified attribute and lang
+     *
+     * @param array $object The object to translate
+     * @param array $included The included translations data
+     * @param string $attribute The attribute name
+     * @param string $lang The lang (2 chars string)
+     * @return string|null
+     */
+    public function exists(array $object, array $included, string $attribute, string $lang) : bool
+    {
+        $val = $this->getTranslatedField($object, $included, $attribute, $lang);
+
+        return ($val !== null);
+    }
+
+    /**
+     * Return translated field per response object and included, attribute and lang. Null on missing translation.
+     * First time that it's called per response object and included, it fills $this->translation data.
+     * I.e.:
+     *
+     *     $this->translation[100]['en'] = ['title' => 'Example', 'description' => 'This is an example']
+     *     $this->translation[100]['it'] = ['title' => 'Esempio', 'description' => 'Questo è un esempio']
+     *     $this->translation[100]['sp'] = ['title' => 'Ejemplo', 'description' => 'Este es un ejemplo']
+     *
+     * @param array $object The object to translate
+     * @param array $included The included translations data
+     * @param string $attribute The attribute name
+     * @param string $lang The lang (2 chars string)
+     * @return string|null The translation of attribute field per object response and lang
+     */
+    private function getTranslatedField(array $object, array $included, string $attribute, string $lang) : ?string
+    {
+        if (empty($object)) {
+            return null;
+        }
+        $id = Hash::get($object, 'id');
+        $path = sprintf('%s.%s.%s', $id, $lang, $attribute);
+        if (!Hash::check($this->translation, $path)) {
+            foreach ($included as $inc) {
```

Comment:
it works, but imposing an indexed `included` structure with type and id like `included.translations.{id}` will surely become easier and faster...
it has cons and pro of course...
@batopa what do you think?